tests/attention/test_trtllm_gen_attention_decode.py

```
@@ -1173,6 +1177,353 @@ def test_trtllm_batch_decode(
+    head_dim: int,
+):
+    """trtllm-gen spec-dec tree decode: q_len_per_req draft tokens per request
+    attend to the prefix plus a tree-structured subset of the draft tail,
+    expressed as a dense (bs, q_len, q_len) bool mask."""
+    compute_capability = get_compute_capability(torch.device(device="cuda"))
+    if compute_capability[0] != 10:
+        pytest.skip("trtllm-gen backend requires SM100 and SM103 GPUs.")
+
+    torch.manual_seed(0)
+    num_qo_heads = num_kv_heads * head_grp_size
+    q_lens, in_kv_lens, seq_lens = generate_seq_lens_decode(
```

Ruff (0.15.20)

[warning] Unpacked variable `in_kv_lens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)